src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -99,7 +99,7 @@ public function read(array $domains, array $locales): TranslatorBag
                 $response = $this->client->request('GET', sprintf('export/locale/%s.xlf', rawurlencode($locale)), [
                     'query' => [
                         'filter' => $domain,
-                        'status' => 'translated',
+                        'status' => 'translated, blank-translation',
```

Comment:
shouldn't we remove the space?

Reply:
Both work, it's up to you

Reply:
please remove it. A space in a query string looks odd

Reply:
Done